src/v/pandaproxy/BUILD

```diff
@@ -123,8 +123,11 @@ redpanda_cc_library(
         "//src/v/config",
         "//src/v/container:fragmented_vector",
         "//src/v/container:json",
+        "//src/v/datalake:schema_avro",
```

Comment:
Can we put these datalake schema libraries in a new directory? Honestly I think they could go in the iceberg directory as a subdirectory too. I think it's important to make it clear it's separate because with these build graph additions datalake has libraries that imports schema registry and now schema registry imports datalake stuff. I think having the converters in another directory makes it clearer it's a lower layer building block.

Reply:
yeah strongly agree. precise location tbd

Reply:
https://github.com/redpanda-data/redpanda/pull/25775